### DISCUSSION TITLE
My Developer Pack application keeps getting rejected without any clear error message.

### DISCUSSION THREAD

**Discussion by ImpetuousJaguarundi** [community/community#65575] [GitHub Education]
### Hi everyone,

Every time I try to apply, the process gets completed till 50%, and then the page reloads, saying "Unfortunately, we weren't able to approve your application." Please help. I have tried uploading a dated receipt from bursar but to no avail.

**Comment by arafajs** [comment]
- Make sure that you are using a school-issued email address.
- Make sure that the email address is verified. You can verify your email address by clicking on the link in the verification email that GitHub sent you.
- Make sure that the date on your receipt is within the current academic year.
- Make sure that the receipt is clear and legible.
- Try uploading a different receipt.
- Try using a different browser.

If you have tried all of these things and you are still having trouble, you can contact GitHub support for help.

Here is the link to the GitHub Student Developer Pack support page:

https://education.github.com/contact
